static/docs/install/index.md

```diff
@@ -16,5 +16,5 @@ Please double check that you don't already have DVC (for example running
 - You can [install a stable pre-release](/doc/install/pre-release) of DVC to
   stay ahead of official releases.
 
-- Please also check out these 3rd-party tool
-  [plugins](/doc/user-guide/install/plugins), which might be useful.
+- Please also check out these 3rd-party tool [plugins](/doc/install/plugins),
```

Comment:
Hmmm this makes me wonder if any others were broken. Let me look up the PR that extracted the install section out of the user guide...

Reply:
Should be all good per the changes in https://github.com/iterative/dvc.org/pull/734/files?file-filters%5B%5D=.json#diff-19166edb015cd737a10d78c350d18229